{"name": "我的代办可以收到用户的申请信息”", "status": "failed", "statusDetails": {"message": "AssertionError: assert '内部处理错误,接口异常' == 'success.'\n  - success.\n  + 内部处理错误,接口异常", "trace": "@allure.epic(\"运营平台\")\n    @allure.feature(\"用户申请\")\n    @allure.title (\"我的代办可以收到用户的申请信息”\")\n    def test_1():\n        #操作账号15222222222\n        false = False\n        true = True\n        null = ''\n        requests.packages.urllib3.disable_warnings()\n        data = {\"userId\": \"464103894448513024\",\n                \"enterpriseName\": \"323\",\n                \"certificateNumber\": \"646546646464646546\",\n                \"name\": \"12321\",\n                \"province\": \"天津市\",\n                \"city\": \"\",\n                \"industry\": \"采矿业/石油和天然气开采业\",\n                'file': ('40293605_1.png', open('C:/test/tp/40293605_1.png', 'rb'), 'image/png', {})\n                }\n        # 1用户发起变更信息操作\n        with allure.step(\"1用户发起变更信息操作\"):\n            res1 = requests.post(URLG + \"customer-center/account/upgrade\",\n                         headers=headers_enterprise, data=data, verify=False, )\n            with allure.step(f\"1.字段msg检查，预期结果为'success'\"):\n                result=jsonpath.jsonpath(json.loads(res1.text),\"$..msg\")[0]\n                with allure.step(f\"实际结果为：\\\"{result}\\\"\"):\n>                   assert result == \"success.\"\nE                   AssertionError: assert '内部处理错误,接口异常' == 'success.'\nE                     - success.\nE                     + 内部处理错误,接口异常\n\ntest_运营侧.py:282: AssertionError"}, "steps": [{"name": "1用户发起变更信息操作", "status": "failed", "statusDetails": {"message": "AssertionError: assert '内部处理错误,接口异常' == 'success.'\n  - success.\n  + 内部处理错误,接口异常\n", "trace": "  File \"E:\\pythonWorkSpace\\api_test_1\\issp\\test_运营侧.py\", line 282, in test_1\n    assert result == \"success.\"\n"}, "steps": [{"name": "1.字段msg检查，预期结果为'success'", "status": "failed", "statusDetails": {"message": "AssertionError: assert '内部处理错误,接口异常' == 'success.'\n  - success.\n  + 内部处理错误,接口异常\n", "trace": "  File \"E:\\pythonWorkSpace\\api_test_1\\issp\\test_运营侧.py\", line 282, in test_1\n    assert result == \"success.\"\n"}, "steps": [{"name": "实际结果为：\"内部处理错误,接口异常\"", "status": "failed", "statusDetails": {"message": "AssertionError: assert '内部处理错误,接口异常' == 'success.'\n  - success.\n  + 内部处理错误,接口异常\n", "trace": "  File \"E:\\pythonWorkSpace\\api_test_1\\issp\\test_运营侧.py\", line 282, in test_1\n    assert result == \"success.\"\n"}, "start": 1694074804756, "stop": 1694074804756}], "start": 1694074804755, "stop": 1694074804758}], "start": 1694074804679, "stop": 1694074804759}], "start": 1694074804679, "stop": 1694074804759, "uuid": "e403b0c8-1c23-4df2-b911-c845f1069494", "historyId": "a572c063cafb298f3b046cc0e3a9e688", "testCaseId": "a572c063cafb298f3b046cc0e3a9e688", "fullName": "test_运营侧#test_1", "labels": [{"name": "epic", "value": "运营平台"}, {"name": "feature", "value": "用户申请"}, {"name": "suite", "value": "test_运营侧"}, {"name": "host", "value": "DESKTOP-ORH7286"}, {"name": "thread", "value": "44332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_运营侧"}]}